DependencyInjection/Configuration.php

```diff
@@ -145,4 +146,26 @@ private function getUploadableNode()
 
         return $node;
     }
+
+    private function getSluggableNode()
+    {
+        $treeBuilder = new TreeBuilder();
+        $node = $treeBuilder->root('sluggable');
+
+        $node
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->arrayNode('managed_filters')
```

Comment:
please add ``->fixXmlConfig('managed_filter')`` on the parent node (i.e. before ``->children()``) to support XML properly for the prototyped node.